minio/xml_marshal.py

```diff
@@ -47,6 +48,72 @@ def xml_marshal_bucket_constraint(region):
     return data.getvalue()
 
 
+def xml_marshal_select(opts):
+    root = s3_xml.Element('SelectObjectContentRequest')
+    expression = s3_xml.SubElement(root, 'Expression')
+    expression.text = opts.expression
+    expression_type = s3_xml.SubElement(root, 'ExpressionType')
+    expression_type.text = 'SQL'
+    input_serialization = s3_xml.SubElement(root, 'InputSerialization')
+
+    if opts.in_ser.csv_input is not None:
+        compression_type = s3_xml.SubElement(input_serialization, 'CompressionType')  
+        compression_type.text = opts.in_ser.compression_type
+        csv = s3_xml.SubElement(input_serialization, 'CSV')
+        file_header_info = s3_xml.SubElement(csv, 'FileHeaderInfo')
+        file_header_info.text = opts.in_ser.csv_input.FileHeaderInfo
+        record_delimiter = s3_xml.SubElement(csv, 'RecordDelimiter')
+        record_delimiter.text = opts.in_ser.csv_input.RecordDelimiter
+        field_delimiter = s3_xml.SubElement(csv, 'FieldDelimiter')
+        field_delimiter.text = opts.in_ser.csv_input.FieldDelimiter
+        quote_charachter = s3_xml.SubElement(csv, 'QuoteCharacter')
```

Comment:
There is a typo here for `quote_character`

Reply:
done